src/client.rs

```diff
@@ -166,7 +150,8 @@ impl Client {
         crate::aio::MultiplexedConnection,
```

Comment:
`fn create_multiplexed_tokio_connection()` config should become `#[cfg(feature = "tokio-rt-core")]`